saltproc/run_saltproc.py

```diff
@@ -11,40 +11,49 @@
 from pyne import serpent
 from pyne import nucname
 
+##############################################################
+######### THIS IS THE INPUT FILE SECTION OF SALTPROC #########
+######### TO RUN, simply set these settings and run  #########
```

Comment:
Comments like this don't generally make their way to the user. It's much better in the readme. Also, a great (and testable) way to make this appear on the command line is to use if __name__ == '__main__' and argparse with a help flag instead.